### PR TITLE
[W4.11e][T12-4]Yang Shuqi

### PR DESCRIPTION
**Pull request by YShu7**

### CODE REVIEW

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -43,4 +43,31 @@ private void assertAreUnique(Object... objects) {
...
+        assertAnyNull('a', null, "a", null);
+
+        // no null items
+        assertNotNull("a", "b");
+        assertNotNull(1, 2, 3);
+        assertNotNull(1, "");
+    }
+
+    private void assertAnyNull(Object... objects) {
+        assertTrue(Utils.isAnyNull(objects));
```
Comment (eugenepeh) [inline test/java/seedu/addressbook/common/UtilsTest.java:67]: Not necessary to abstract the `assertTrue(Utils.isAnyNull(...))` as separate method, because asserting the result of the method itself is fine.

Although existing test `assertAreUnique` does that, but that is to prevent `Arrays.asList(...)` from flying everywhere.

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 2]
```diff
@@ -43,4 +43,31 @@ private void assertAreUnique(Object... objects) {
...
+        // empty list
+        assertNotNull();
+
+        // 1 null items
+        assertAnyNull((Object) null);
+
+        // some null items
+        assertAnyNull(null, null);
+        assertAnyNull(null, 1,null);
+        assertAnyNull('a', null, "a", null);
```
Comment (eugenepeh) [inline test/java/seedu/addressbook/common/UtilsTest.java:58]: Sometimes when adding test cases, think about how the method treat this new test case differently from the other existing ones.

**Review by eugenepeh — COMMENTED** [review]
Good job on adding a variety of test cases (both positive and negative test cases, as well as empty and null scenario).

Some comments added. Please close the PR after reading the comments.
